packages/synapse-interface/constants/chains/master.tsx

```diff
@@ -128,7 +128,7 @@ export const CANTO: Chain = {
     primary: 'https://mainnode.plexnode.org:8545',
     fallback: 'https://canto.slingshot.finance',
   },
-  nativeCurrency: { name: 'Canto', symbol: 'CANTO', decimals: 18 },
+  nativeCurrency: { name: 'Canto', symbol: 'NOTE', decimals: 18 },
```

Comment:
CANTO is the gas token of the Canto network, not NOTE:

https://docs.canto.io/technical-reference/token-economics